File: Assets/CS_Scripts/InstructionsMenuGUI.cs
using UnityEngine;
using System.Collections;

public class InstructionsMenuGUI : MonoBehaviour {
	public GUISkin menuSkin;
	public float areaWidth;
	public float areaHeight;

	void OnGUI() {
		GUI.skin = menuSkin;
		float ScreenX = ((Screen.width * 0.5f) - (areaWidth * 0.5f));
		float ScreenY = ((Screen.height * 0.7f) - (areaHeight * 0.5f));
		
		GUILayout.BeginArea (new Rect (ScreenX, ScreenY, areaWidth, areaHeight));
		if (GUILayout.Button("Play")) // creates and listens to mouse press
			OpenLevel("Game");
		if (GUILayout.Button("Quit"))
			Application.Quit();
		
		GUILayout.EndArea();
	}

	void OpenLevel(string level) {	
		Wait();
		Application.LoadLevel(level);
	}	
	
	IEnumerator Wait() {
		yield return new WaitForSeconds(0.35f);
	}
		
}

File: Assets/Scripts/InstructionsMenuGUI.js
var menuSkin:GUISkin;
var areaWidth:float;
var areaHeight:float;

function OnGUI() {
	GUI.skin = menuSkin;
	var ScreenX = ((Screen.width * 0.5) - (areaWidth * 0.5));
	var ScreenY = ((Screen.height * 0.7) - (areaHeight * 0.5));
	
	GUILayout.BeginArea (Rect (ScreenX, ScreenY, areaWidth, areaHeight));
	if (GUILayout.Button("Play")) // creates and listens to mouse press
		OpenLevel("Game");
	if (GUILayout.Button("Quit"))
		Application.Quit();
	
	GUILayout.EndArea();
}

function OpenLevel(level:String) {	
	yield new WaitForSeconds(0.35);
	Application.LoadLevel(level);
}

@script RequireComponent(AudioSource)